Formula/automysqlbackup.rb

```diff
@@ -56,4 +58,7 @@ def plist; <<-EOS.undent
     </plist>
     EOS
   end
+
+  test do
```

Comment:
There is no `--version` switch available for `automysqlbackup` 

- The default configuration file has a version number in its first line but that is previous version number, so can't be used to write a test for current version. 
- Running `automysqlbackup` without any options returns a long output which contains the version number, provided `mysql` is already installed. Can I use this for writing an `assert_match` to match version using regex ? But I guess, this may not be a good test as the output might change and this test might fail.